css-three-column-layout/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container align-items-center justify-content-center">
+    <div class="row justify-content-center">
+      <div class="column-full">
+        <div class="header">
+          <h2>Designed for life science. Optimized for SaMD and Digital Health.</h2>
+        </div>
+        <div class="row">
+          <div class="column-third align-items-center justify-content-center">
+            <section class="align-items-center">
+              <div class="circle align-items-center"><img src="images/merge.png"></div>
+              <h3>Streamlined QMS</h3>
+              <p class="opaque">A QMS that includes modules for all stages of the product life cycle from Design
+                Control to CAPA
+              </p>
+            </section>
+          </div>
+          <div class="column-third align-items-center justify-content-center">
+            <section class="align-items-center">
+              <div class="circle align-items-center"><img src="images/cloud-computing.png"></div>
+              <h3>Integrates with existing tools</h3>
+              <p class="opaque">Enzyme QMS integrates with you existing tools, such as JIRA, GitHub and Jenkins with Enzyme</p>
+            </section>
+          </div>
+          <div class="column-third align-items-center justify-content-center">
+            <section class="align-items-center">
+              <div class="circle align-items-center"><img src="images/discussion.png"></div>
+              <h3>In-depth Onboarding</h3>
+              <p class="opaque">We train your entire team and provide you with support when and as it's needed</p>
+            </section>
+          </div>
+        </div>
+      </div>
```

Comment:
This `row` is inside of the above `row`, make sure it is a sibling to that `row`